overlays/custom-packages/element-desktop/default.nix

```diff
@@ -0,0 +1,7 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
```

Comment:
We are at 2024 now

Reply:
> We are at 2024 now

True, the modifications were made last year. 
I tought that there is script to update the the copy write headers, instead of manually update every year.

Reply:
Modified as proposed